lib/sass/components/Modal/sass/Modal.scss

```diff
@@ -39,6 +41,50 @@
       margin-top: 0;
     }
   }
+
+  &__theme-header {
+    position: absolute;
+    top: -1px;
+    left: 0;
+    right: 0;
+    height: var(--theme-header-height);
+    border-top-left-radius: $general-border-radius;
+    border-top-right-radius: $general-border-radius;
+
+    .tyk-icon {
+      position: absolute;
+      left: calc(50% - var(--theme-header-icon-size) / 2);
+      top: calc((var(--theme-header-icon-size) - var(--theme-header-height)) / -2);
+      width: var(--theme-header-icon-size);
+      height: var(--theme-header-icon-size);
+      border-radius: 50%;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      font-size: text-scale('xxl');
+      color: white;
```

Comment:
are we not using a var for this one ?

Reply:
If you're thinking of using the `$white` variable I don't think it makes sense.
If you're thinking something like creating a variable for it (e.g. `$modal-icon-color`) at the moment it is not needed. We can create a variable for it when the need to overwrite it arises.